kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -142,7 +142,7 @@ grCollectionsPanel.controller('GrNodeCtrl',
     }));
 
     ctrl.isSelected = ctrl.selectedCollections.some(col => {
-        return angular.equals(col, ctrl.node.data.fullPath);
+        return angular.equals(col, ctrl.node.data.data.pathId.split('/'));
```

Comment:
Out of curiosity, why is `selectedCollections` split by `/` and not just the `pathId` string? Having to split the `pathId` here adds cost to the watch expression for every single node.